doc/release-notes-31384.md

```diff
@@ -0,0 +1,17 @@
+- Node and Mining
+
+---
+
+- PR #31384 fixed an issue where the coinbase transaction weight was being reserved in two different places.
+  The fix ensures that the reservation happens in a single place.
+
```

Comment:
You lost a line here:

```md
- This PR also introduces a new startup option, `-maxcoinbaseweight` (default: `8,000` weight units).
```

Reply:
Fixed in latest push https://github.com/bitcoin/bitcoin/compare/7452638559e8ccbe00db5ef5a53cb21ffe27d337..2fb833e69627a8f9b1c4b14b60d1df9903adf210